svgdigitizer/svgplot.py

```diff
@@ -336,10 +572,35 @@ def axis_label(axis):
             return labels[-1]
 
         return {
-            self.xlabel: axis_label(self.xlabel),
-            self.ylabel: axis_label(self.ylabel),
+            axis_symbol: axis_label(axis_symbol) for axis_symbol in self.axis_symbols
         }
 
+    @property
+    @cache
+    def _grouped_ref_points(self):
+        r"""
+        Return the reference points grouped by axis symbol.
+        """
+
+        # group ref_points by axis
+        keyfunc = lambda x: str(x.label).split(":")[0][
+            :-1
+        ]  # allow multi character axis label
+        ref_points = [points[0] for points in self.labeled_paths["ref_point"]]
+        # without sorting only the last contiguous occurences are returned in the grouping dict
+        ref_points = sorted(ref_points, key=keyfunc)
+        # grouping by first letter of the label text
+        grouped_ref_points = {
+            quantity: list(g) for quantity, g in groupby(ref_points, keyfunc)
```

Comment:
```suggestion
            variable: list(g) for variable, g in groupby(ref_points, keyfunc)
```

I think it is more generic to call this the variable of the plot.

Reply:
True.